src/main.rs

```diff
@@ -171,17 +171,9 @@ fn build_hyperfine_options<'a>(
     }
 
     match (min_runs, max_runs) {
-        (Some(min), _) if min < 2 => {
-            // We need at least two runs to compute a variance.
-            return Err(OptionsError::RunsBelowTwo);
```

Comment:
The `RunsBelowTwo` enum entry should be removed, I guess. Normally, the compiler should warn about this, but the OptionsError is public (should really be `pub(crate)`), so the compiler assumes this might be used by someone "outside"

Reply:
Fixed, thanks. No idea those were a thing, more Rust beginner mistakes...

Reply:
Fixed, thanks!